src/main/scala/nl/knaw/dans/easy/export/package.scala

```diff
@@ -25,12 +25,17 @@ import scala.xml._
 
 package object export {
 
-  def invert[T1,T2] (m: Map[T1,T2]): Map[T2,T1] =
-    m.map{case (key,value) => (value,key)}
+  def invert[T1, T2](m: Map[T1, T2]): Map[T2, T1] =
+    m.map { case (key, value) => (value, key) }
 
   def toSdoName(objectId: String): String =
     objectId.replaceAll("[^0-9a-zA-Z]", "_")
 
+  implicit class RichNode(n: Node) {
+    def headOfAttr(namespace: String, attrName: String): Option[String] =
+      n.attribute(namespace, attrName).flatMap(xs => xs.headOption.map(_.text))
+  }
+
```

Comment:
a review note:

I used the package object for all kinds of convenience methods. The implicit classes add these convenience methods to classes of external libraries. Having these implicit classes in the package object, they are in-scope everywhere in the package without an explicit import, thus they become automatically available for code-completion. It might be a disadvantage that the object thus becomes dependent on many libraries what might make reorganizing packages more complicated.